src/js/apps/patients/worklist/worklist_app.js

```diff
@@ -258,6 +266,25 @@ export default App.extend({
 
     Radio.request('alert', 'show:success', renderTemplate(BulkEditActionsSuccessTemplate, { itemCount }));
   },
+  showCountView() {
+    if (!this.collection || !this.filteredCollection.length) {
+      this.hideCountView();
+      return;
+    }
+
+    const countView = new CountView({
+      isFlowList: this.getState().isFlowType(),
+      count: this.filteredCollection.length,
+      maximumCount: MAXIMUM_LIST_COUNT,
+      isFindInListApplied: this.getState('searchQuery').length > 2,
+      hasReachedMaximum: this.collection.length === MAXIMUM_LIST_COUNT,
```

Comment:
The capped messaging will only show if the list count is exactly equal to the maximum count (i.e. `500`). Which is based on the `MAXIMUM_LIST_COUNT` static variable

We'll have to keep the maximum list count in-sync with what the backend is using.

Would it be a good idea to have the actions/flows API request include a `maxCount: 500` item or something similar?

Than we wouldn't have to update `MAXIMUM_LIST_COUNT` manually if the backend changes the maximum count on their end.